app/constants/index.js

```diff
@@ -51,3 +46,14 @@ export const mediaURL = contentApiURL + '/media'
 export const tagsURL = contentApiURL + '/tags'
 
 export const customScheduleURL = contentApiURL + '/pages/3679'
+
+export const donateURL = contentApiURL + '/pages/2'
+export const membershipURL = contentApiURL + '/pages/1425'
+
+export const membershipStripeURL = contentApiURL + '/pages/4980'
+export const membershipStripeCancelURL = contentApiURL + '/pages/4992'
+export const membershipStripeThanksURL = contentApiURL + '/pages/4994'
+
+export const donateStripeURL = contentApiURL + '/pages/5088'
```

Comment:
Can we refer to these pages via the page **title** in the URL? Like this it's hardly testable (we cannot guarantee that we have the same page number on the dev server).

Reply:
Never mind! Page ID-s are editable in WP!